src/js/edu-benefits/utils/validations.js

```diff
@@ -322,8 +322,7 @@ function isValidRotcScholarshipAmount(data) {
 }
 
 function isValidRotcHistoryPage(data) {
-  return data.seniorRotcCommissioned.value !== 'Y' || (isNotBlank(data.seniorRotc.commissionYear.value)
-    && data.seniorRotc.rotcScholarshipAmounts.every(isValidRotcScholarshipAmount));
+  return data.seniorRotcCommissioned.value !== 'Y' && data.seniorRotc.rotcScholarshipAmounts.every(isValidRotcScholarshipAmount);
```

Comment:
This should be `||` not `&&`. Either they weren't commissioned through ROTC or they were in which case the scholarship amounts need to be valid.